package/MDAnalysis/coordinates/PDB.py

```diff
@@ -364,6 +375,21 @@ def _read_next_timestep(self, ts=None):
         return self._read_frame(frame)
 
     def _read_frame(self, frame):
+        """
+        Read frame from PDB file.
+
+        Notes
+        -----
+        When the CRYST1_ record has unitary values (cubic box with sides of 1Å),
+        cell dimensions are considered fictitious. An user warning is raised
+        and cell dimensions are not set.
```

Comment:
Can we be more precise here or are we waiting for a solution for #2698  ?

Reply:
I added the following since for the PDB reader cell dimensions are given by `_init_unitcell` in `Timestep`

> cell dimensions are set to :code:`np.array([0.0, 0.0, 0.0, 0.0, 0.0, 0.0])` (see Issue #2698)